src/pages/PullRequestPage/subroute/CommitsTab/shared/CIStatus/CIStatus.tsx

```diff
@@ -29,9 +32,15 @@ Status.propTypes = {
   ciPassed: PropTypes.bool,
 }
 
-function CIStatus({ commitid, coverage, ciPassed }) {
+interface CIStatusProps {
+  commitid?: string
+  coverage?: number
+  ciPassed?: boolean | null
+}
+function CIStatus({ commitid, coverage, ciPassed = false }: CIStatusProps) {
   return (
     <div className="flex-1 flex-col items-end lg:flex">
+      {/* @ts-ignore */}
```

Comment:
we can just do `hook="commitid-link" isExternal={false}` instead of ignoring ts warnings